substrateinterface/contracts.py

```diff
@@ -446,11 +446,12 @@ def __init__(self, *args, **kwargs):
         """
         self.__contract_events = None
         self.contract_metadata = kwargs.pop('contract_metadata')
+        self.contract_address = kwargs.pop('contract_address')
         super(ContractExecutionReceipt, self).__init__(*args, **kwargs)
 
     @classmethod
     def create_from_extrinsic_receipt(cls, receipt: ExtrinsicReceipt,
-                                      contract_metadata: ContractMetadata) -> "ContractExecutionReceipt":
+                                      contract_metadata: ContractMetadata, contract_address: str) -> "ContractExecutionReceipt":
```

Comment:
Better to make `contract_address` optional, otherwise it wouldn't be backwards compatible and can't be released as a minor fix.